app/src/ereferrals/java/org/eyeseetea/malariacare/domain/boundary/repositories/ISettingsRepository.java

```diff
@@ -1,7 +1,12 @@
 package org.eyeseetea.malariacare.domain.boundary.repositories;
 
+import org.eyeseetea.malariacare.domain.entity.MediaListMode;
 import org.eyeseetea.malariacare.domain.entity.Settings;
 
 public interface ISettingsRepository {
     Settings getSettings();
+
+    MediaListMode getMediaListMode();
+
+    void saveMediaListMode(MediaListMode mediaListMode);
```

Comment:
This method is not correct because this data source should receive to save only settings domain entity